sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -69,11 +66,7 @@ export const ContentPlacementContainerStyles = css`
         }
     }
 
-    @media screen and (min-width: 1000px) {
-        :host {
-            --flow: column;
```

Comment:
I believe deleting this variable's value assignment(--flow) is what is causing your issue.